app/components/header/HeaderMain.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Image from 'next/image';  
+
+const HeaderMain: React.FC = () => {
+    return (
+        <header className="flex justify-between items-center p-4 text-white relative bg-cover" style={{ backgroundImage: 'url(/images/headerbg.svg)' }}>
+            <div className="flex items-center">
```

Comment:
This flex box is unnecessary, remove the div and change the justify in the div above.